### PR TITLE
Graph class command

### PR DESCRIPTION
**Pull request by Abdulrahman-Aladdin**

### CODE REVIEW

File: src/main/java/commandLine/CLI.java [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+    public static void clearScreen () {
+        for (int i = 0; i < 100; i++) System.out.println();
+    }
+
+    private void firstMenu () {
+        System.out.println("1- SSSP");
+        System.out.println("2- APSP");
+        System.out.println("3- Check negative cycles");
+        System.out.println("4- Exit");
```
Comment (hussainmansour) [inline src/main/java/commandLine/CLI.java]: what is SSSP and APSP meaning?

File: src/main/java/org/example/Graph.java [hunk 2]
```diff
@@ -1,30 +1,87 @@
...
-    int[][] floydWarshall();
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class Graph implements IGraph{
+    private double[][] graphM;
+    private int V, E;
+    private ArrayList<ArrayList<Integer>> graphL;
```
Comment (hussainmansour) [inline src/main/java/org/example/Graph.java]: Dijkstra want ArrayList<ArrayList<Pair<NodeName,Weight>>>
NodeName --> Integer
Weight --> Double

**Comment by Abdulrahman-Aladdin** [comment]
check the changes

File: src/main/java/commandLine/CLI.java [hunk 3]
```diff
@@ -134,8 +134,8 @@ public static void clearScreen () {
     }
 
     private void firstMenu () {
-        System.out.println("1- SSSP");
-        System.out.println("2- APSP");
+        System.out.println("1- SSSP"); // Single Source Shortest Path
+        System.out.println("2- APSP"); // All Pairs Shortest Paths
```
Comment (hussainmansour) [inline src/main/java/commandLine/CLI.java]: great jop for all, but i think it a good practice if you print it in the menu as the user may get confused.

File: src/main/java/org/example/PairOfThree.java [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package org.example;
+
+public class PairOfThree extends Pair{
+    int src;
```
Comment (hussainmansour) [inline src/main/java/org/example/PairOfThree.java]: name it triple

**Review by hussainmansour — APPROVED** [review]

**Review by MohamedAnwar121 — APPROVED** [review]

**Pull request merged**